### ISSUE TITLE
Monitor segfaults on Fedora

### ISSUE THREAD

**Issue by electricduck** [stsdc/monitor#352]
This seems to have started happening after the last build of Monitor on Copr ([#5564634](https://copr.fedorainfracloud.org/coprs/stsdc/monitor/build/5564634/)); on trying to launch Monitor, the indicators briefly appear and then it segfaults...

```
ducky@tanager:/tmp$ com.github.stsdc.monitor 

 Monitor 0.16.0 

Segmentation fault (core dumped)
```

However, on another machine with the exact same package (and I checked with `rpm -qi com.github.stsdc.monitor` &mdash; both show the exact same build dates), it works fine, however, that machine is slightly out-of-date with other packages. So it seems to me this is some odd environment problem **or** there's a misbehaving dependency.

**Comment by stsdc** [comment]
Published 0.16.1 for Fedora. Please check.

**Comment by electricduck** [comment]
Seems to be working once again!
